app/lib/ui/views/account/account_editing_screen.dart

```diff
@@ -69,6 +69,38 @@ class _AccountEditingScreenState extends State<AccountEditingScreen> {
                               color: NavyBlue,
                               fontWeight: FontWeight.bold),
                         ),
+                        Padding(
+                          padding: EdgeInsets.only(
+                            top: 20.0,
+                          ),
+                        ),
+                        CircleAvatar(
+                          radius: 53,
+                          backgroundColor: NavyBlue,
+                          child: model.selectedImage != null
+                              ? ClipRRect(
+                                  borderRadius: BorderRadius.circular(50),
+                                  child: Image.file(
+                                    model.selectedImage,
```

Comment:
- [ ] This image could be null, you need to check this and put something else when it is